Gruntfile.js

```diff
@@ -88,18 +88,34 @@ module.exports = function(grunt) {
 			dryrun: { options: {service: "qa", dryRun: true} },
 			qa: { options: {service: "qa"} },
 			prod: { options: {service: "prod"} }
+		},
+		"shell": {
+			"deployrumlambda": {
+				command: env => {
+
+					// These AWS profiles are expected to be in ~/.aws/credentials
```

Comment:
Do you have an example of what these should be? Should we store them in LastPass?